boost-maven/boost-boms/booster-mp14-bom/pom.xml

```diff
@@ -25,17 +25,27 @@
             <dependency>
                 <groupId>boost.boosters</groupId>
                 <artifactId>boost-mp14-apis-bom</artifactId>
-                <version>0.1-SNAPSHOT</version>
+                <version>1.0-M1-SNAPSHOT</version>
                 <scope>import</scope>
                 <type>pom</type>
             </dependency>
             <dependency>
                 <groupId>boost.boosters</groupId>
                 <artifactId>ee8-bom</artifactId>
-                <version>0.1.3-SNAPSHOT</version>
+                <version>1.0-M1-SNAPSHOT</version>
```

Comment:
You didn't cause this issue.. but MP 1.4 correlates to EE 7, not 8

Reply:
What does it say about our IT coverage that we don't catch this?      Is this actually not taking precedence and so is just harmless noise?   Or do we not really care if we get EE 8 level features since they are back compatible with EE 7 test apps?   

Those are kind of open-ended, so let me ask a simpler question:  do we every use the mp14-bom in any IT?    Might we at least add one IT to do that here?

Reply:
switched to ee7 - and switched the mpHealth IT to use the mp14 bom